benchmarks/analyzeNode/analyze_node_test.go

```diff
@@ -17,7 +17,7 @@ func runBenchmarkAnalyzeNode(b *testing.B, pathName string) {
 	runtime.GC()
 
 	for i := 0; i < b.N; i++ {
-		node.Analyze("testNode", pathName, node.AnalyzeOpts{UncertifiedRHEL: false, IsRHCOSRequired: true})
+		node.Analyze("testNode", pathName, node.AnalyzeOpts{UncertifiedRHEL: false, IsRHCOSRequired: false})
```

Comment:
curious why this changed

Reply:
If IsRHCOSRequired is true, it won't do the analyze on non-core os node like ubi8 or 9